apps/extension/src/routes/popup/settings/settings.tsx

```diff
@@ -1,55 +1,73 @@
 import {
-  GlobeIcon,
+  DashboardIcon,
+  ExitIcon,
+  Link1Icon,
   LockClosedIcon,
-  MixerHorizontalIcon,
+  Share1Icon,
   TextAlignLeftIcon,
 } from '@radix-ui/react-icons';
-import { PopupPath } from '../paths';
-import { FadeTransition } from '../../../components';
-import { BackIcon, Button } from 'ui/components';
+import { CustomLink, FadeTransition, SettingsHeader } from '../../../shared';
+import { useStore } from '../../../state';
+import { passwordSelector } from '../../../state/password';
 import { usePopupNav } from '../../../utils/navigate';
+import { PopupPath } from '../paths';
 
 const links = [
   {
     title: 'Advanced',
-    icon: <MixerHorizontalIcon className='h-5 w-5 text-foreground' />,
+    icon: <DashboardIcon className='h-5 w-5 text-muted-foreground' />,
     href: PopupPath.SETTINGS_ADVANCED,
   },
   {
     title: 'Security & Privacy',
-    icon: <LockClosedIcon className='h-5 w-5 text-foreground' />,
+    icon: <LockClosedIcon className='h-5 w-5 text-muted-foreground' />,
     href: PopupPath.SETTINGS_SECURITY,
   },
+  {
+    title: 'Permission',
+    icon: <TextAlignLeftIcon className='h-5 w-5 text-muted-foreground' />,
+    href: PopupPath.SETTINGS_PERMISSION,
+  },
   {
     title: 'Networks',
-    icon: <GlobeIcon className='h-5 w-5 text-foreground' />,
+    icon: <Share1Icon className='h-5 w-5 text-muted-foreground' />,
     href: PopupPath.SETTINGS_NETWORKS,
   },
   {
-    title: 'Permission',
-    icon: <TextAlignLeftIcon className='h-5 w-5 text-foreground' />,
-    href: PopupPath.SETTINGS_PERMISSION,
+    title: 'Connected sites',
+    icon: <Link1Icon className='h-5 w-5 text-muted-foreground' />,
+    href: PopupPath.SETTINGS_CONNECTED_SITES,
   },
 ];
 
 export const Settings = () => {
   const navigate = usePopupNav();
+  const { clearSessionPassword } = useStore(passwordSelector);
+
   return (
-    <FadeTransition className='flex flex-col items-stretch justify-start'>
-      <BackIcon className='absolute top-6 text-foreground' onClick={() => navigate(-1)} />
-      <h1 className='border-b border-[rgba(75,75,75,0.50)] pb-2 pt-5 text-center'>Settings</h1>
-      <div className='flex flex-1 flex-col items-start gap-4 p-4'>
-        {links.map(i => (
-          <Button
-            key={i.href}
-            variant='ghost'
-            className='flex w-full items-center justify-start gap-2 p-[10px] text-left hover:bg-transparent hover:opacity-50'
-            onClick={() => navigate(i.href)}
-          >
-            {i.icon}
-            <p className='text-foreground'>{i.title}</p>
-          </Button>
-        ))}
+    <FadeTransition>
+      <div className='flex min-h-[100vh] w-[100vw] flex-col justify-between gap-6'>
+        <SettingsHeader title='Settings' />
+        <div className='flex flex-1 flex-col items-start gap-5 px-[30px]'>
```

Comment:
Thes icons need to be vertically centered with the icon next to it so they are the same height.

<img width="203" alt="Screenshot 2023-09-20 at 2 30 44 PM" src="https://github.com/penumbra-zone/web/assets/16624263/66cddb54-f27c-46be-a143-4124fbc72dcf">

Reply:
it`s not clear for me

Reply:
Advanced it to high. I needs to be in the middle. It's the same for the rest as well.

<img width="203" alt="269282671-66cddb54-f27c-46be-a143-4124fbc72dcf" src="https://github.com/penumbra-zone/web/assets/16624263/d4679a62-b122-4cd3-893e-85ac037a8656">